src/services/StreamService.ts

```diff
@@ -75,22 +76,23 @@ class StreamService {
         Object.assign(templateRecord, {});
         console.log("before for each");
         if (record.testTypes instanceof Array) {
-              record.testTypes?.forEach((testType: any, i: number, array: any[]) => {
-                  console.log("in for each");
-                  const clonedRecord: any = Object.assign({}, templateRecord); // Create record from template
-                  Object.assign(clonedRecord, { testTypes: testType }); // Assign it the test type
-                  Object.assign(clonedRecord, {
-                      // Assign certificate order number
-                      order: {
-                          current: i + 1,
-                          total: array.length,
-                      },
-                  });
-
-                  splittedRecords.push(clonedRecord);
+          record.testTypes?.forEach(
+            (testType: any, i: number, array: any[]) => {
+              console.log("in for each");
```

Comment:
I can see this was there before, but this very much feels like a debug-type log. Are we getting value from it being there?

Reply:
Yeah honestly thought the same thing, I left it in because I assumed somebody left it there for a reason for a certain type of error and we aren't losing much by it being there. Happy to pull it out tho

Reply:
I'm assuming the CloudWatch logs are presumably littered with this which was my main gripe. Any preference to retain or are we happy to clean this up? @cb-cs @m-coslett

Reply:
Happy to clean this up, we will shortly be moving this logic into cvs-cert-gen with the EventBridge implementation anyway.

Reply:
https://github.com/dvsa/cvs-tsk-cert-gen-init/pull/75/files -> found this, we signed this off in May and it was tested as an AC like this is something we wanted for a reason, so I am going to keep it in.